src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+@use '../utils/variables.scss' as variables;
```

Comment:
Ensure that the file '../utils/variables.scss' exists and contains the necessary variables like `$day-gap`, `$days`, `$day-size`, `$day-bg`, `$border-size`, `$font-size`, `$font-color`, and `$hover-bg`.